api/src/services/mapSearch/mapSearch.ts

```diff
@@ -8,15 +8,16 @@ import {
 
 export const searchNearby: QueryResolvers['searchNearby'] = async ({
   location,
-  radius,
+  // radius,
```

Comment:
instead of this, let's make it optional on the SDL and conditionally either put that in the options or rankby